src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -18,6 +12,12 @@
 using System.Linq;
 using System.Reflection;
 using System.Runtime.CompilerServices;
+using Microsoft.CodeAnalysis.Collections;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
```

Comment:
Is there a meaningful change here? If not, please revert

Reply:
Had to disable auto code cleanup on save since this is not typical using directives order)